api/src/server.rs

```diff
@@ -312,6 +320,27 @@ where
             model,
             insert_task,
             marker: PhantomData,
+            feature_flags: HashSet::default(),
+        }
+    }
+
+    pub fn enable_feature(&mut self, feature: &str) {
```

Comment:
longer term I image feature flags living inside the service crate or similar so they can be shared by recon and other parts of the system but it's implementation details and fine for now

Reply:
Yeah, there's better crates for feature flags, we may want to do a quick survey and find one that meets our needs before this or other features are done.